fleet-provisioning/src/entry.c

```diff
@@ -187,8 +191,98 @@ static GglError fetch_from_db(FleetProvArgs *args) {
     return GGL_ERR_OK;
 }
 
-GglError run_fleet_prov(FleetProvArgs *args) {
-    GglError ret = fetch_from_db(args);
+static GglError update_cred_access(void) {
+    char *args[] = { "chown", "-R", "ggcore:ggcore", "/ggcredentials/", NULL };
```

Comment:
While ggcore is our default, it is configurable by customer with GGL_SYSTEMD_SYSTEM_USER. So this doesn't work on any install that has configured the user to be something different.

Reply:
if they are building it from the scratch then they will need to update this section also as there is no way for me to pull in cmake  configurations

Reply:
Sounds good, maybe we can bring this out to be a macro so if they do want to configure this it's not buried too deep in the code

Reply:
Added a warning in the docs